dropwizard-health/src/main/java/io/dropwizard/health/DefaultHealthFactory.java

```diff
@@ -61,8 +66,13 @@
     private List<String> healthCheckUrlPaths = ImmutableList.of("/health-check");
```

Comment:
Would it make sense to add a constant for this path based on `DEFAULT_BASE_NAME`?

Reply:
Definitely!